lib/active_model/serializer/adapter.rb

```diff
@@ -1,12 +1,14 @@
 module ActiveModel
   class Serializer
     class Adapter
+      UnknownAdapterError = Class.new(ArgumentError)
```

Comment:
> 
They're both pretty common for exceptions, and is pretty much required for structs.  That the poro fixtures do it, though, I think is problematic 
>